### PR TITLE
LESS cleanup

### PR DESCRIPTION
**Pull request by FatBoyXPC**
Updated all the CSS to more appropriate nested LESS, moved the variables from various files into ccm-variables.import.less. Used some parent selector tricks.

The navbar retains current styling as close as possible to match the "old" way the CSS was. I disabled the collapse killing CSS and changed the grid-float-breakpoint variable (which is the "proper" way to change when collapse happens).

I had an idea about the navbar, though. I think the gears icon on mobile would look better on the line below: [current](http://i.imgur.com/tjVF6ok.png) vs [proposed](http://i.imgur.com/cJe8pHc.png). I left that out of the current PR because I feel like this is out of scope for a LESS cleanup. I'd gladly add that, or create an issue on it and tackle it in another PR.

### CODE REVIEW

**Comment by jfly** [comment]
First off, thanks for doing this, James!

I haven't looked into the code yet, but moving the gear icon down to a new line looks good to me, lets definitely do it! Up to you if you want to do it as part of this PR, or a separate one.

File: css/ccm-variables.import.less [hunk 1]
```diff
@@ -0,0 +1,67 @@
+////////////////////////////////////////////////////////////////////////////////
+// Bootstrap overrides
+////////////////////////////////////////////////////////////////////////////////
+
+// We want to prevent the navbar from collapsing
+@grid-float-breakpoint: 0px;
+
+// Bootstraps default table-bg-hover is too similar to the darker
```
Comment (jfly) [inline css/ccm-variables.import.less]: nitpick: possesive, so use an apostrophe in "Bootstraps"

**Comment by jfly** [comment]
I just looked through the diff, it looks great to me!

On comment, though: I' was surprised to find the navbar logic split into 2 places: ccm-variables.import.less and layout.import.less. How do you feel about pulling it out of those 2 places and putting it in a ccm-navbar.import.less or something?

**Comment by FatBoyXPC** [comment]
I committed some changes per your comments :)

**Comment by jfly** [comment]
:package: IT UP AND SHIP IT

**Pull request merged**